index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books- Es6</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <nav>
+        <h1>Awesome Books</h1>
+        <ul id="menu">
+            <li id="list-link" >
+                <a href="#list-section">List</a>
+            </li>
+            <li id="book-link">
+                <a href="#book-section">Add Book</a>
+            </li>
+            <li id="contact-link">
+                <a href="#contact-section">Contact</a>
+            </li>
+        </ul>
```

Comment:
Great design @binodbhusal 🤝

- [ ] Kindly ensure each navigation link has its own independent content. The `List` should have a list of books and delete buttons, `Add Book` should contain two inputs, and the Add Book button, and `Contact` should contain the contact information. You may fix this issue since for your design all links contain all the information.
![navlinks](https://user-images.githubusercontent.com/63917725/235552153-ffab5479-e717-458d-883f-d83556b2d2b6.PNG)

Reply:
- [ ] Great work implementing this change 👏🏾